### ISSUE TITLE
Cell editable by default

### ISSUE THREAD

**Issue by vitortmdiniz** [AllenFang/react-bootstrap-table#1065]
I want to make a cell from the table editable by default, so the user won't need to click or double click to trig it. Is there anyway i can do it? Thanks!

**Comment by AllenFang** [comment]
@vitortmdiniz, sorry, we dont have this feature, but I can support this in the future, but without schedule now(I'm busy here).

If I want to support this feature, there's some question:
1. How many cells can be edited by default, one or more?
2. After save, how to perform the cell? still as editable?

If you are hurry to have this feature, I suggest you to use `dataFormat` to make your cell as an editor and call the API to save the data into table, there're some people doing this in the past.

What's your choice?

**Comment by vitortmdiniz** [comment]
Hey, sorry for take long time to answer. My intention would be that all cells from a specifical row would be editable. And yes, after they would still as editable. 
I'm doing like you said, using dataFormat to create an input field in the cell, the problem with that is that is necessary to click in the input field to make it 'editable'. If you use Tab for example to navigate from cell to cell it works like a read-only field. Do you know how to fix it? Thanks!

**Comment by AllenFang** [comment]
@vitortmdiniz, I think the default input field was disable and become a normal input field after user click on it?! Am I right?

**Comment by vitortmdiniz** [comment]
@AllenFang Yes, exactly! is it possible to make it active without need to click on it?

**Comment by AllenFang** [comment]
> is it possible to make it active without need to click on it?

so how did you make the input field become to enable from disabled?

**Comment by supiash1** [comment]
+1 for feature

**Comment by kksandyrox** [comment]
Greetings @AllenFang , I stumbled upon a situation, that required me to have something similar to this. Is this feature out? Or are you planning to release this soon?

**Comment by prateek04** [comment]
I also have the same requirement.Rather than click on the cell. One column having cell with input field by default. You help will be appreciated.
